src/diffusers/utils/peft_utils.py

```diff
@@ -169,14 +169,18 @@ def get_peft_kwargs(rank_dict, network_alpha_dict, peft_state_dict, is_unet=True
             lora_alpha = set(network_alpha_dict.values()).pop()
 
     # layer names without the Diffusers specific
-    target_modules = list({name.split(".lora")[0] for name in peft_state_dict.keys()})
+    target_modules = list(
+        {name.split(".lora")[0] for name in peft_state_dict.keys()}
+    )  # if "lora_magnitude_vector" not in name
+    use_dora = any("lora_magnitude_vector" in k for k in peft_state_dict)
```

Comment:
since `use_dora` is only present on peft>=0.9.0 do we need to somehow guard here the peft version and pop `use_dora` from `lora_config_kwargs` if peft version is lower than 0.9.0?

Reply:
Or raise an error inviting users to upgrade PEFT

Reply:
100 percent. This should be the case! Will fix.

Reply:
Done in da29b91f.